.github/workflows/mirror-php-package.yml

```
@@ -0,0 +1,68 @@
+        run: |
+          chmod +x "./packages/php/${{ github.event.inputs.package }}/tasks/mirror-prepare.sh"
+          ./packages/php/${{ github.event.inputs.package }}/tasks/mirror-prepare.sh "mirror-working-dir/woocommerce/${{ github.event.inputs.package }}"
+  
+      - name: Push to mirror
+        uses: Automattic/action-push-to-mirrors@v2.2.2
+        with:
+          token: ${{ secrets.API_TOKEN_GITHUB }}
+          username: matticbot
+          working-directory: ${{ github.workspace }}/mirror-working-dir
+          commit-message: "Mirror package ${{ github.event.inputs.package }} from WooCommerce"
+        timeout-minutes: 5 # Normally should take up to 30 seconds
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)